src/Test/contlinear.jl

```diff
@@ -411,6 +411,11 @@ function linear2test(model::MOI.ModelLike, config::TestConfig)
             @test MOI.get(model, MOI.ConstraintDual(), vc1) ≈ 0 atol=atol rtol=rtol
             @test MOI.get(model, MOI.ConstraintDual(), vc2) ≈ 1 atol=atol rtol=rtol
         end
+        if config.basis
```

Comment:
Nit: There's a newline before the `if config.duals` but not before `if config.basis`. I don't have a preference but this should be consistent at least.